_maps/map_files/IceBoxStation/IceBoxStation.dmm

```diff
@@ -30396,7 +30397,6 @@
 /obj/structure/table/optable{
 	name = "Robotics Operating Table"
 	},
-/obj/item/surgical_drapes,
```

Comment:
this right here is the change in this PR that most offends me. why would you choose the remove the DRAPES here? they can be replaced entirely by a bedsheet from dorms (or even maintenance itself). literally all this change does is make it slightly more inconvenient to do surgery in maint robotics if you forgot to pick up a bedsheet while going through your powergamer checklist and make it much harder for players who don't know that you can use bedsheets in the place of surgical sheets to do actual surgery in maint robotics.

in short, you're actively _discouraging_ people from using maint robotics for actual maint surgery and such, and instead encouraging people to just treat it as a source of minor loot.

why would someone leave a pile of surgical tools here but not surgical drapes, anyway?

Reply:
doing surgery in maint is supposed to be inconvenient

Reply:
bedsheets are a direct upgrade from drapes because you can put them on your neck